### PR TITLE
Add and update some companies

### PR DESCRIPTION
**Pull request by NoriSte**
I added some companies I recently got interviewed from.

Thanks for the amazing project ❤️

### CODE REVIEW

**Review by edoardocostantinidev — CHANGES_REQUESTED** [review]
Thanks a lot @NoriSte for your contributions! Just added a small request change (we usually use the linkedin career page as a backup for `career_page_url`). Also just want to hear @alessandromr on the .json formatting style, I don't think it's an issue for our generator, but since we have followed a somewhat standard json format I would like to know his opinion about it.

This collection is only made possible by contributors such as yourself, so thanks and hearts goes to you!

File: data/capbase.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Capbase",
+  "url": "https://capbase.com/",
+  "career_page_url": "",
+  "type": "Product",
+  "categories": ["startup"],
```
Comment (edoardocostantinidev) [inline data/capbase.json:6]: Also allowed categories are the following -> 

> Allowed Company Categories
> - cloud_software
> - design_ux
> - marketing_writing
> - hr

**Review by edoardocostantinidev — CHANGES_REQUESTED** [review]

File: data/capbase.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Capbase",
+  "url": "https://capbase.com/",
+  "career_page_url": "",
```
[the remark this answers is not in the document]
Comment (NoriSte) [reply data/capbase.json]: Thanks, @edoardocostantinidev , I've just push-forced the commit that now includes this change

File: data/capbase.json (hunk 1 above)
Comment (NoriSte) [reply data/capbase.json]: Sorry for the mistake! Fixed

**Review by alessandromr — APPROVED** [review]

**Pull request merged**